libs/hdf-converters/sample_jsons/gosec_mapper/sample_input_report/Grype_gosec_results.json

```diff
@@ -0,0 +1,124 @@
+{
+	"Golang errors": {},
+	"Issues": [
+		{
+			"severity": "MDIUM",
```

Comment:
why is medium spelled wrong here?  did this typo come from the original source?

Reply:
changed to MEDIUM, i must have accidentally hit a button